config/routes.rb

```diff
@@ -14,10 +14,12 @@
   devise_for :users
 
   root :to => 'home#index'
-
-  get '/auth/:provider/callback' => 'authentications#create'
+  get '/auth/:provider/callback', to: 'session#create'
+  get '/auth/failure', to: redirect('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.